securedrop/source_templates/base.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="utf-8">
     <title>SecureDrop | Protecting Journalists and Sources</title>
-    {% assets filters="cssmin", output="gen/source.css",
-      "css/normalize.css", "css/source.css", "css/font-awesome.css" %}
+
+    {# TODO why doesn't a link to /static/source.css work? #}
```

Comment:
We should resolve this TODO before merging. It's best not to leave TODOs in merged code, if possible.